cmd/pippi/frontend/src/app/strings/strings.component.html

```diff
@@ -0,0 +1,10 @@
+<ace-editor
```

Comment:
Cool, this is a useful HTML tag/component. Was it part of ace originally, or did you hook it up?